x-pack/plugins/reporting/server/browsers/chromium/driver_factory/puppeteer_launcher.ts

```diff
@@ -5,7 +5,7 @@
  * 2.0.
  */
 
-import puppeteer, { type PuppeteerLaunchOptions } from 'puppeteer';
+import puppeteer, { type LaunchOptions as PuppeteerLaunchOptions } from 'puppeteer';
```

Comment:
only structural difference from any of the backports - name change to a puppeteer type